### ISSUE TITLE
list() should be renamed to something else

### ISSUE THREAD

**Issue by yokawasa** [dreamfactorysoftware/azure-documentdb-php-sdk#1]
In PHP, list() is restricted word, and the following list() function should be renamed, otherwise parse error will occur
```
Parse error: parse error, expecting `"identifier (T_STRING)"' in azure-documentdb-php-sdk/src/Contracts/ResourceInterface.php on line XX
```
## list functions
- https://github.com/dreamfactorysoftware/azure-documentdb-php-sdk/blob/master/src/Contracts/ResourceInterface.php#L17
- https://github.com/dreamfactorysoftware/azure-documentdb-php-sdk/blob/master/src/Resources/Database.php#L31
- https://github.com/dreamfactorysoftware/azure-documentdb-php-sdk/blob/master/src/Resources/Collection.php#L37
- https://github.com/dreamfactorysoftware/azure-documentdb-php-sdk/blob/master/src/Resources/Document.php#L42

For restricted words in PHP, see http://php.net/manual/en/reserved.keywords.php

**Comment by yokawasa** [comment]
ping ping

**Comment by a15lam** [comment]
We will address this issue in upcoming release. In the meantime you can use PHP 7+ to void this error.

**Comment by df-arif** [comment]
We will address this issue in upcoming release. In the meantime you can use PHP 7+ to void this error.

**Comment by leehicks** [comment]
Fixed in version [0.2.0](https://github.com/dreamfactorysoftware/azure-documentdb-php-sdk/releases/tag/0.2.0). Used in latest version of df-azure [0.10.1](https://github.com/dreamfactorysoftware/df-azure/releases/tag/0.10.1)

**Comment by yokawasa** [comment]
@df-arif @leehicks  Thank you so much for your prompt response!  Most of all, thanks for a great SDK!
